src/HybridSolver.cpp

```diff
@@ -855,7 +855,7 @@ void HybridSolver::BuildParallelSystemAndSolver()
         else
         {
             prec_ = make_unique<mfem::HypreBoomerAMG>(*pHybridSystem_);
-            ((mfem::HypreBoomerAMG&)*prec_).SetPrintLevel(0);
+            static_cast<mfem::HypreBoomerAMG&>(*prec_).SetPrintLevel(0);
```

Comment:
@gelever @barker29 Is `static_cast` more preferred here?

Reply:
[CppCoreGuidelines](https://github.com/isocpp/CppCoreGuidelines/blob/d49c720e98a914de4a21e6b8618dd4102c0a0139/CppCoreGuidelines.md#c146-use-dynamic_cast-where-class-hierarchy-navigation-is-unavoidable) recommends dynamic cast even if you know the type it will be.

Reply:
I took the liberty of fixing this, removing the cast entirely. If you don't like it I'll revert.